src/gs-details-page.c

```diff
@@ -2413,7 +2418,7 @@ gs_details_page_app_copied (GsPage *page, GsApp *app, const GError *error)
 {
 	GsDetailsPage *self = GS_DETAILS_PAGE (page);
 
-	if (error == NULL) {
+	if (error == NULL && g_strcmp0 (gs_app_get_id (self->app), gs_app_get_id (app)) == 0) {
```

Comment:
Also would be better to do a pointer comparison as in `gs_details_page_plugin_status_changed_cb` (for performance and for correctness since `gs_app_get_id()` doesn't uniquely identify an app).